content/applications/websites/ecommerce/ecommerce_management/customer_accounts.rst

```diff
@@ -0,0 +1,47 @@
+=================
+Customer accounts
+=================
+
+Having customer accounts on an eCommerce allows customers to access all their documents in a single
+place. To access their account, customers must be **logged-in** on the eCommerce website and under
+their **account name**, click :guilabel:`My Account`. From there, customers have access to their
+:guilabel:`quotations`, :guilabel:`orders`, :guilabel:`invoices`, etc.
```

Comment:
Having customer accounts on eCommerce allows customers to access all their documents in a single
place. To access their account, customers must be **logged-in** on the eCommerce website and, under
their **account name**, click :guilabel:`My Account`. From there, customers can access their :guilabel:`quotations`, :guilabel:`orders`, :guilabel:`invoices`, etc.

Reply:
@chiaraprattico can you put these comments in suggestions so it's easier to find the differences?